src/org/exist/xquery/parser/XQueryTree.g

```diff
@@ -470,7 +470,7 @@ throws PermissionDeniedException, EXistException, XPathException
 			{
 				PathExpr enclosed= new PathExpr(context);
 				SequenceType type= null;
-				QName qn = QName.parse(staticContext, qname.getText());
+				QName qn = QName.parse(staticContext, qname.getText(), null);
```

Comment:
it's same as [other](https://github.com/shabanovd/exist/blob/5e93f51c0d0a0d699179ddac22e0712500b56517/src/org/exist/xquery/parser/XQueryTree.g#L1476) [places](https://github.com/shabanovd/exist/blob/5e93f51c0d0a0d699179ddac22e0712500b56517/src/org/exist/xquery/parser/XQueryTree.g#L1967) and [expected](https://github.com/eXist-db/exist/blob/005045fda2ee0df0c3871b12d1b90a79f086ffbc/src/org/exist/dom/QName.java#L330)